src/VisualStudio/LiveShare/Impl/Client/RemoteLanguageServiceWorkspace.cs

```diff
@@ -192,10 +196,15 @@ public Document GetOrAddDocument(string filePath)
                 return null;
             }
 
+            if (_remoteRootPath == null)
```

Comment:
I'm curious - what is the user experience like when we open an empty workspace with this change, then start adding things?  I'm not sure exactly what will work vs won't, but I'd expect a bunch to not work (far, gotodef / razor).

Reply:
Yup, that's right. This scenario (empty->non-empty workspace) has many other bugs aside from language services being in a semi-broken state, so I think it's OK to punt that issue a little further down the road.

Reply:
Can you add a comment pointing to a followup issue for resolving the language service issue?

Reply:
Issue [here](https://github.com/dotnet/roslyn/issues/39475).

Reply:
(Feel free to assign to me)